README.md

```diff
@@ -141,6 +141,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 
 * [enginesound](https://github.com/DasEtwas/enginesound) — A GUI and command line application used to procedurally generate semi-realistic engine sounds. Featuring in-depth configuration, variable sample rate and a frequency analysis window.
 * [indiscipline/zrtstr](https://github.com/indiscipline/zrtstr) — A command line utility for checking if stereo wav files are faux-stereo (i.e. have identical channels) and converting such files to mono. [<img src="https://api.travis-ci.org/indiscipline/zrtstr.svg?branch=master">](https://travis-ci.org/indiscipline/zrtstr)
+* [Lyriek](https://github.com/bartwillems/lyriek) — A multi-threaded GTK 3 application for fetching the lyrics of the current playing song. <img src=https://github.com/bartwillems/lyriek/workflows/Rust/badge.svg>
```

Comment:
Please correct the build badge syntax to use markdown syntax.

Reply:
Fixed

Reply:
Now that it's using the Markdown syntax, the build has picked up the missing branch parameter. Please see https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter and add the branch parameter for the main/master branch of the repository